### ISSUE TITLE
Download with file dialog

### ISSUE THREAD

**Issue by dotWAVE** [FlorentUguet/QBooru#12]
There should be an option when downloading a file to save it with the file dialog rather than just going into the "downloads" folder.

**Comment by FuSoftware** [comment]
Good idea, gonna put it in the V2 as I keep working on it. Sounds like a better idea than the current one. Since I'm not supporting the 1.0 anymore, I will not add it in it, but as I said, it'll be in 2.0
